CMakeLists.txt

```
@@ -78,8 +78,16 @@ else()
   endif()
 
     # Look for installed Torchvision package in lib paths
-  if(TRITON_PYTORCH_ENABLE_TORCHVISION AND NOT EXISTS "${TRITON_PYTORCH_LIB_PATHS}/libtorchvision.so.1")
-    message(WARNING "TRITON_PYTORCH_ENABLE_TORCHVISION is on, but TRITON_PYTORCH_LIB_PATHS does not contain Torchvision package")
+  if(TRITON_PYTORCH_ENABLE_TORCHVISION)
+    if(${RHEL_BUILD})
+      if(NOT EXISTS "${TRITON_PYTORCH_LIB_PATHS}/libtorchvision.so")
+        message(WARNING "TRITON_PYTORCH_ENABLE_TORCHVISION is on, but TRITON_PYTORCH_LIB_PATHS does not contain Torchvision package")
+      endif()
+    else()
+      if(NOT EXISTS "${TRITON_PYTORCH_LIB_PATHS}/libtorchvision.so.1")
+        message(WARNING "TRITON_PYTORCH_ENABLE_TORCHVISION is on, but TRITON_PYTORCH_LIB_PATHS does not contain Torchvision package")
+      endif()
+    endif()
```

Original statement wasn't written right. Expanding that statement in such a way may be not the best way. 
Instead it should be:
```suggestion
  find_library( LIBTORCHVISION libtorchvision.so libtorchvision.so.1 PATHS ${TRITON_PYTORCH_LIB_PATHS} )
  if(NOT ${LIBTORCHVISION}) 
    message(WARNING "TRITON_PYTORCH_ENABLE_TORCHVISION is on, but TRITON_PYTORCH_LIB_PATHS does not contain Torchvision package")
  endif(NOT ${LIBTORCHVISION})
```